src/WindowsPackageManager/ConfigurationStaticFunctions.cpp

```diff
@@ -22,11 +24,32 @@ namespace ConfigurationShim
     {
     };
 
+    AppInstaller::Logging::Level ConvertLevel(winrt::Microsoft::Management::Configuration::DiagnosticLevel level)
+    {
+        switch (level)
+        {
+        case winrt::Microsoft::Management::Configuration::DiagnosticLevel::Verbose: return AppInstaller::Logging::Level::Verbose;
+        case winrt::Microsoft::Management::Configuration::DiagnosticLevel::Informational: return AppInstaller::Logging::Level::Info;
+        case winrt::Microsoft::Management::Configuration::DiagnosticLevel::Warning: return AppInstaller::Logging::Level::Warning;
+        case winrt::Microsoft::Management::Configuration::DiagnosticLevel::Error: return AppInstaller::Logging::Level::Error;
+        case winrt::Microsoft::Management::Configuration::DiagnosticLevel::Critical: return AppInstaller::Logging::Level::Crit;
+        }
+
+        return AppInstaller::Logging::Level::Info;
+    }
+
     struct
     DECLSPEC_UUID(WINGET_OUTOFPROC_COM_CLSID_ConfigurationStaticFunctions)
     ConfigurationStaticFunctionsShim : winrt::implements<ConfigurationStaticFunctionsShim, winrt::Microsoft::Management::Configuration::IConfigurationStatics>
     {
-        ConfigurationStaticFunctionsShim() = default;
+        ConfigurationStaticFunctionsShim()
+        {
+            auto threadGlobalsRestore = m_threadGlobals.SetForCurrentThread();
+            auto& diagnosticsLogger = m_threadGlobals.GetDiagnosticLogger();
+            diagnosticsLogger.EnableChannel(AppInstaller::Logging::Channel::All);
+            diagnosticsLogger.SetLevel(AppInstaller::Logging::Level::Verbose);

```

Comment:
Do we need to call ConvertLevel above to set corresponding level? The ConvertLevel does not seem to be called in this change.